tgui/packages/tgui/interfaces/CircuitAccessChecker.tsx

```diff
@@ -28,8 +28,8 @@ export const CircuitAccessChecker = (props) => {
           </LabeledList.Item>
         </LabeledList>
         <AccessConfig
-          accesses={regions}
-          selectedList={accesses}
+          accesses={regions || []}
+          selectedList={accesses || []}
```

Comment:
these should already be casted in L16. what's going on here?

Reply:
Apparently not, it just bluescreens.
It's possible that those are getting overriden by the fact that these values may be null, not undefined.
Idk if react will take values if null is passed in, I know it won't if it's undefined.